### PR TITLE
UI features like extra buttons for navigation and text for user experience

### PR DESCRIPTION
**Pull request by Capn05**
This PR has alot of code and files changed but it is all just css styling, moving components around for better aesthetics, and adding navigation buttons, all for a better user experience. 

Screenshot 2024-07-30 at 3 04 18 PM

### CODE REVIEW

File: src/Modal.jsx [hunk 1]
```diff
@@ -43,38 +44,64 @@ function CreateModal({ isOpen, close, movie }) {
 
   if (!isOpen) return null;
   if (!movieDetails) return <div>Loading...</div>;
-
   return (
-    <div className="modal-overlay">
-      <div className="modal-content">
-        <button className="modalClose" onClick={close}>
-          X
+    <section className="modal-overlay" onClick={close}>
```
Comment (BriptimusPrimus) [inline src/Modal.jsx:48]: Good job adding semantically expressive tags

File: src/MovieList.jsx [hunk 2]
```diff
@@ -63,7 +64,7 @@ function MovieList({
   const Cards = data?.results?.slice(0, AMOUNT_MOVIES).map((movie, index) => (
     <MovieCard
       key={movie.id}
-      movieImage={`https://image.tmdb.org/t/p/w342${movie?.poster_path}`}
+      movieImage={`https://image.tmdb.org/t/p/w780${movie?.poster_path}`}
```
Comment (BriptimusPrimus) [inline src/MovieList.jsx:67]: What's the difference between w342 and w780?

Comment (Capn05) [reply src/MovieList.jsx:67]: The resolution of the image

File: src/HomePage.jsx [hunk 3]
```diff
@@ -58,9 +61,20 @@ const HomePage = () => {
       .then((response) => setGenres(response));
   }, []);
 
+  useEffect(() => {
+    if (currentUser) {
+      fetch(`http://localhost:3000/users/userMovies/${currentUser.uid}`)
+        .then((response) => response.json())
+        .then((data) => {
+          console.log(data.length);
```
[the remark this answers is not in the document]
Comment (Capn05) [reply src/HomePage.jsx:69]: my mistake!

[the remark this answers is not in the document]
Comment (BriptimusPrimus) [reply src/HomePage.jsx:69]: It's all good

**Pull request merged**